### ISSUE TITLE
"PosterFrame" html artifact when mp has no "presenter/player+preview"

### ISSUE THREAD

**Issue by spiraleddy** [polimediaupv/paella#172]
Hi, Our live streams mediapackages are not currently published with a "presenter/player+preview" attachment which makes the PosterFrame shows ALT text on the screen.  Could the 04_video_container  include a check for the existence of data.preview for the cases where the preview is missing? 
Than you, Karen

paella/src/04_video_container.js
line 824
		`if (this._showPosterFrame) {
			videoNode.setPosterFrame(data.preview); <--- data.preview may not exist
		}`

Possible protection:
		`if (this._showPosterFrame && data.preview) { 
			videoNode.setPosterFrame(data.preview); 
		}`
